src/assets/scss/main.scss

```diff
@@ -197,3 +222,45 @@ a.desc:after {
 .summary {
     margin-bottom: $paragraph-margin-bottom;
 }
+
+
+table {
+    h3 {
+        margin: 0.5em 0;
+    }
+}
+
+.phpinfo {
+    h1, h2 {
+        text-align: center;
+    }
+}
+.config-php-info-table {
+    tr {
+        &.h {
+            background-color: #99c!important;;
```

Comment:
Isn't it possible doing that w/o `!important`? Making selector more specific?

Reply:
You caught my one "oh, screw that" moment.
Fixed